pymc3/distributions/bart.py

```diff
@@ -257,6 +278,24 @@ def rvs(self):
                     return i
 
 
+class NormalSampler:
+    def __init__(self, sigma):
+        self.size = 5000
```

Comment:
Why 5000?

Reply:
larger cache size does not seem to provide any speed up.